uzi_vc_issuer/ura_issuer.go

```diff
@@ -48,63 +52,61 @@ func NewUraVcBuilder(didCreator did_x509.DidCreator, chainParser x509_cert.Chain
 
 // Issue generates a URA Verifiable Credential using provided certificate, signing key, subject DID, and subject name.
 func (u DefaultUraIssuer) Issue(certificateFile string, signingKeyFile string, subjectDID string, test bool) (string, error) {
-	reader := pem2.NewPemReader()
-	certificate, err := reader.ParseFileOrPath(certificateFile, "CERTIFICATE")
+	pemReader := pem2.NewPemReader()
+	pemBlocks, err := pemReader.ParseFileOrPath(certificateFile, "CERTIFICATE")
 	if err != nil {
 		return "", err
 	}
-	_certificates, err := u.chainParser.ParseCertificates(certificate)
-	if err != nil {
-		return "", err
-	}
-	if len(*_certificates) != 1 {
-		err = fmt.Errorf("did not find exactly one certificate in file %s", certificateFile)
-		return "", err
-	}
 
-	chain, err := ca_certs.GetDERs(test)
-	if err != nil {
-		return "", err
+	if len(pemBlocks) == 1 {
```

Comment:
allows the pem file from the command line to contain the cert chain instead of only the signing cert.